src/routes.json

```
@@ -114,6 +114,10 @@
     {
       "name": "require-billing-modal",
       "component": "requirePaymentModal"
+    },
+    {
+      "name":"ListItem-delete-confirmation-modal",
```

Typically, the name doesn’t start with a capital letter
```suggestion
      "name":”delete-line-item-confirmation-modal",
```